packages/@okta/vuepress-site/guides/sign-into-spa/section03/react/config.md

```diff
@@ -1,2 +1,71 @@
-In your application's `App.js` file, import the following objects and pass in your configuration:
-{code}
\ No newline at end of file
+Configure a Security component in your top-level component, 'App.jsx'. The Security component will create a child context which exposes the Auth object to child components.
+
+```javascript
+
+
+import React, { Component } from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Security } from '@okta/okta-react';
+
+
+const config = {
+  clientId: '{clientId}',
+  issuer: 'https://{yourOktaDomain}.com/oauth2/default',
+  redirectUri: 'http://localhost:8080/implicit/callback',
+};
+
+
+class App extends Component {
+  render() {
+    return (
+      <Router>
+        <Security {...config} >
+          { /* App routes go here */ }
+        </Security>
+      </Router>
+    );
+  }
+}
+
+export default App;
+```
+
+If you find that your application needs to interact with the Auth object outside of a child component context, you can create the Auth object directly and pass it to the Security component as shown below.
```

Comment:
If you find that your application needs to interact with the Auth object outside of a child component context, you can create the Auth object directly and pass it to the Security component.

I'm removing references to "as shown below" - I know they were in the content originally, but it's not good practice for accessibility requirements in content, so I'm removing them as we go. It's clear by the next sentence with the colon what is being represented "below".